### DISCUSSION TITLE
What is the name of the ancient road that connected southern 🇮🇹?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#809] [Galaxy Trivia] [answered]
**Bonus:** What cities did it connect? How long was it?

**Comment by ParanoidUser** [comment]
@rakshitmanishmalhotra / @rsouza01, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment, marked as answer]
The **Appian Way** (Latin and Italian: **Via Appia**) is one of the earliest and strategically most important Roman roads of the ancient republic. It **connected Rome to Brindisi**, in southeast Italy.

The Appian Way was constructed in 312 BC and served as a crucial route for both military and civilian purposes, facilitating trade, transportation, and communication throughout the Roman Empire.

The exact length of the Appian Way is difficult to determine, as the road underwent expansions and modifications over time. However, the **original stretch from Rome to Capua was approximately 230 kilometers (143 miles) long**. Over time, it was **extended further south, eventually reaching a total length of around 540 kilometers (335 miles) when it reached Brindisi**.

**Comment by SANKARAMDAS** [comment]
The Appian Way was a Roman road used as a main route for military supplies for its conquest of southern Italy in 312 BC and for improvements in communication. The Appian Way was begun in 312 bce by the censor Appius Claudius Caecus.
Appian Way, Latin Via Appia, the first and most famous of the ancient Roman-road-system, running from Rome  to Campania-region-Italy and southern Italy. The Appian Way was begun in 312 BCE by the censor-ancient-Roman-official Appius Claudius Caecus . At first it ran only 132 miles (212 km) from Rome south-southeastward to ancient Capua-ancient-city-Italy, in Campania, but by about 244 BCE it had been extended another 230 miles (370 km) southeastward to reach the port of Brundisium (Brindisi), situated in the “heel” of Italy and lying along the Adriatic-Sea.

https://www.britannica.com/topic/Appian-Way
